src/test/java/uk/gov/ons/census/casesvc/messaging/RmNonComplianceReceiverTest.java

```diff
@@ -174,4 +218,53 @@ public void testFieldCordAndOfficerIdsUpdated() {
             any(),
             eq(expectedDateTime));
   }
+
+  @Test
+  public void testFieldCordAndOfficerIdsUpdatedForFinalWarningLetter() {
```

Comment:
This unit test looks identical to `testReceiveMessageWithNoFieldOfficerOrCordUpdatesNCFW`